<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConditionEditor</class>
 <widget class="QDialog" name="ConditionEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="verticalSpacing">
    <number>4</number>
   </property>
   <item row="2" column="0" rowspan="4" colspan="2">
    <widget class="QGroupBox" name="cond_edit_box">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="0" column="1">
       <widget class="QLineEdit" name="var"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="value"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Var:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="cast_combobox">
        <item>
         <property name="text">
          <string>int</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>float</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="cast_checkbox">
        <property name="text">
         <string>Cast: </string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Value: </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Operation: </string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="operation">
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>==</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>!=</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&gt;</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&lt;</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&gt;=</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&lt;=</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>or</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>and</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nor</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nand</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="3" rowspan="5">
    <widget class="QGroupBox" name="data_box">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QListWidget" name="data">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
